Userland/Applications/Browser/Tab.cpp

```diff
@@ -38,8 +39,14 @@
 
 namespace Browser {
 
+static String s_search_engine_format = {};
+
 URL url_from_user_input(const String& input)
 {
+    if (!input.starts_with("?") && !s_search_engine_format.is_null()) {
```

Comment:
Logic error:

```suggestion
    if (input.starts_with("?") && !s_search_engine_format.is_null()) {
```

(please make sure to test your PR again when making changes!)

Reply:
Fixed.